package.json

```diff
@@ -16,11 +16,11 @@
     "eslint-config-airbnb-base": "^11.1.1",
     "eslint-plugin-import": "^2.2.0",
     "eslint-plugin-mocha": "^4.9.0",
-    "html-webpack-plugin": "^2.28.0",
+    "html-webpack-plugin": "webpack-contrib/html-webpack-plugin",
```

Comment:
Why?

Reply:
The HTML webpack plugin owner is currently on holiday (or something) and this is the webpack 4 compatible version until it gets merged into the main fork.

Reply:
Update: looks like the fork is merged now, I'll update this version.